esp/services/ws_workunits/ws_wudetails.cpp

```diff
@@ -0,0 +1,349 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2017 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+#include "ws_workunitsService.hpp"
+#include "ws_wudetails.hpp"
+
+void CWUDetails::process(IEspWUDetailsResponse &resp)
+{
+    unsigned __int64 maxTimestamp = 0;
+    IArrayOf<CAttributeFilter> arrayAttributeFilter;
+    UnsignedShortArray skReturnAttribList;
```

Comment:
Initially, I was storing an Array of StatisticKind. Didn't change after changing the type of array.  Anyway, I've removed the prefix now.